java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -189,7 +189,7 @@
     <rhn-tab-url>/rhn/manager/systems/details/virtualization/guests/${sid}</rhn-tab-url>
     <rhn-tab name="Guests" url="/rhn/manager/systems/details/virtualization/guests/${sid}"/>
     <rhn-tab name="Storage" url="/rhn/manager/systems/details/virtualization/storage/${sid}" acl="system_has_salt_entitlement()"/>
-    <rhn-tab name="Provisioning" url="/rhn/systems/details/virtualization/ProvisionVirtualizationWizard.do" acl="system_feature(ftr_kickstart) or system_feature(ftr_snapshotting)"/>
+    <rhn-tab name="Provisioning" url="/rhn/systems/details/virtualization/ProvisionVirtualizationWizard.do" acl="system_has_management_entitlement(); system_feature(ftr_kickstart) or system_feature(ftr_snapshotting)"/>
```

Comment:
Not sure if this is the best way. I think correct would be to add ftr_kickstart feature to salt management.

Reply:
it was has "management" entitlement. So this is ok